### ISSUE TITLE
has sails-postgresql been audited?

### ISSUE THREAD

**Issue by kevinburkeshyp** [balderdashy/sails-postgresql#144]
If it has passed a security audit, would appreciate a link to the results.

**Comment by tjwebb** [comment]
> If it has passed a security audit, would appreciate a link to the results.

Do you have a specific security test in mind that you'd like run?

**Comment by kevinburkeshyp** [comment]
Sure, I'd like a professional security engineer to 1) examine the codebase for SQL injection opportunities, 2) pen-test a sample app (or a live app) and check for SQL injection opportunities. I would feel much better about the possibility of not leaking the database if these things were complete.

**Comment by sailsbot** [comment]
Thanks for posting, @kevinburkeshyp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
